py/py_cromwell_setup.py

```diff
@@ -30,15 +30,16 @@ def check_for_app(env):
             if workspace_namespace == env['workspace_namespace']:
                 return potential_app_name, potential_app_status, proxy_url['cromwell-service']
 
-    return None, None
+    return None, None, None
 
 def get_app_details(env, app_name):
     get_app_url = f'{env["leonardo_url"]}/api/google/v1/apps/{env["google_project"]}/{app_name}'
     print('start')
     r = requests.get(
         get_app_url,
         params={
-            'includeDeleted': 'true'
+            'includeDeleted': 'true',
+            'role': 'creator'
```

Comment:
does this additionally filter it to only my user's apps?

Reply:
yes, ticket ref: https://broadworkbench.atlassian.net/browse/IA-3823